CONTRIBUTING.md

```diff
@@ -35,6 +35,28 @@ Schemas are only changed _after_ the specification is changed.  Changes are made
 
 * Issue #3576: [What is our authoritative spec URL and how do people find it?](https://github.com/OAI/OpenAPI-Specification/issues/3576)
 
+## Style Guide
+
+Contributions to this repository should follow the style guide as described in this section.
+
+### Markdown
+
+Markdown files in this project should follow the style enforced by the [markdownlint tool][],
```

Comment:
```suggestion
Markdown files in this project should follow the style enforced by the [markdownlint tool](https://www.npmjs.com/package/markdownlint),
```

Do we want to
```sh
npm i -D markdownlint markdownlint-cli
```
and add the updated package.json and package-lock.json to this PR?